code/game/atoms.dm

```diff
@@ -563,6 +563,25 @@
 					for(var/datum/reagent/R in reagents.reagent_list)
 						total_volume += R.volume
 					. += "[total_volume] units of various reagents"
+				//-------- Beer goggles ---------
+				if(user.can_see_boozepower())
+					var/total_boozepower = 0
+					var/list/taste_list = list()
+					if(length(reagents.reagent_list))
+						for(var/datum/reagent/consumable/ethanol/B in reagents.reagent_list)
+							var/real_boozepower = B.boozepwr
+							if(real_boozepower<0) // minus booze power is reversed to light drinkers, but is actually 0 to normal drinkers.
```

Comment:
minor nitpick but add spacing for me pls

Reply:
done